### ISSUE TITLE
Please expose resource_policies as a variable for the instance_template

### ISSUE THREAD

**Issue by ryboe** [terraform-google-modules/terraform-google-vm#315]
### TL;DR

The `google_compute_instance_template` resource supports adding a [`resource_policy`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_template#resource_policies), but it's not exposed as a variable in this module. I would like to set a policy that the instance is restarted weekly. Please add something like this to `variables.tf` so we can set resource policies.

```terraform
variable "resource_policies" {
  type = list(string)
}
```

### Terraform Resources

1. [`google_compute_instance_template.resource_policies`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_template#resource_policies)
2. [`google_compute_resource_policy`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_resource_policy)

### Detailed design

```terraform
module "instance_template" {
  source  = "terraform-google-modules/vm/google//modules/instance_template"
  ...
  resource_policies = [ google_compute_resource_policy.sunday_night_restart.self_link ]
}

resource "google_compute_resource_policy" "sunday_night_restart" {
  name   = "sunday-night-restart"
  region = "us-central1"
  description = "Sunday night restart"
  instance_schedule_policy {
    vm_start_schedule {
      schedule = "30 23 * * 7"
    }
    vm_stop_schedule {
      schedule = "0 23 * * 7"
    }
    time_zone = "US/Central"
  }
}
```
